models/public/f3net/f3net.md

```diff
@@ -0,0 +1,96 @@
+# f3net
+
+## Use Case and High-Level Description
+
+F3Net: Fusion, Feedback and Focus for Salient Object Detection. For details see
+the [repository](https://github.com/weijun88/F3Net), [paper](https://arxiv.org/abs/1911.11445)
+
+## Specification
+
+| Metric                          | Value                                    |
+|---------------------------------|------------------------------------------|
+| Type                            | Salient object detection                 |
+| GFLOPs                          | 31.2883                                  |
+| MParams                         | 25.2791                                  |
+| Source framework                | PyTorch\*                                |
+
+## Accuracy
+
+| Metric    | Value |
+| --------- | ----- |
+| F-measure | 84.21%|
+
+The F-measure estimated on Pascal-S dataset and defined as the harmonic mean of precision and recall.
```

Comment:
Could you provide some link?

Reply:
provided more details and link on dataset